test/Order.spec.ts

```diff
@@ -0,0 +1,142 @@
+import { Format, Order, Protected, Template } from "../src";
+import { client } from "./client";
+import { sleep } from "../src/utils";
+import { IOrder } from "../src/models/_interfaces/IOrder";
+
+let order: Order = null as unknown as Order;
+let template: Template = null as unknown as Template;
+
+beforeAll(async function () {
+  template = await client.createTemplate({
+    name: `Test Order ${new Date().toISOString().replaceAll(":", "-")}`,
```

Comment:
Uh, I don't think this produces a valid ISO string anymore. Are you sure our API will be able to parse this?